### PR TITLE
Rd 9303 otel js distro add version printing when we init the tracer

### PR DESCRIPTION
**Pull request by shanishiri**

### CODE REVIEW

File: src/wrapper.test.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import * as fs from 'fs';
 import * as utils from './utils';
+import { logger } from './utils';
```
Comment (mmanciop) [inline src/wrapper.test.js:3]: Why do we need this line? We import '*' in the one above

File: src/wrapper.ts [hunk 2]
```diff
@@ -16,6 +16,7 @@ import LumigoMongoDBInstrumentation from './instrumentations/mongodb/MongoDBInst
 import { extractEnvVars, getMaxSize, isEnvVarTrue, logger } from './utils';
 import * as awsResourceDetectors from '@opentelemetry/resource-detector-aws';
 import { AwsEcsDetector, LumigoDistroDetector } from './resources/detectors';
+import packageJson from '../package.json';
```
Comment (mmanciop) [inline src/wrapper.ts:19]: We are actually already reading this data in the detector that sets lumigo.distro.version. Cannot we read the version just once?

**Review by mmanciop — CHANGES_REQUESTED** [review]

File: src/wrapper.ts (hunk 2 above)
Comment (shanishiri) [reply src/wrapper.ts]: yes we can!

**Review by mmanciop — APPROVED** [review]

**Review by saartochner — APPROVED** [review]
Awesome!
Your test is not strong because it will keep passing even when you move the file (which will break the behavior). But I don't know how to do it better, so I approve 👍

**Pull request merged**